src/routes/safe/components/Layout.jsx

```diff
@@ -92,6 +91,15 @@ const Layout = (props: Props) => {
   const { address, ethBalance, name } = safe
   const etherScanLink = getEtherScanLink('address', address)
 
+  const onNotificationClosesCb = () => {
```

Comment:
our router is synced with redux so this can be moved out from the component

Reply:
done